service/routes.py

```diff
@@ -106,3 +139,13 @@ def check_content_type(media_type):
         status.HTTP_415_UNSUPPORTED_MEDIA_TYPE,
         f"Content-Type must be {media_type}",
     )
+
+def find_wishlist_by_id(wishlist_id):
+    """Finds wishlist by its ID"""
+    try:
+        wishlist = Wishlist.query.filter_by(id=wishlist_id).first()
+        return wishlist
+    except SQLAlchemyError as e:
+        app.logger.error("Error while retrieving Wishlist by ID: %s", e)
```

Comment:
We could use a different log level, maybe info or debug in this case because it's not really an error, just an end user operation with invalid input.  (Or rather valid input but no resulting record).